### DISCUSSION TITLE
Customizable icon for launchers?

### DISCUSSION THREAD

**Discussion by bijucyborg** [trungleduc/jupyter_app_launcher#33] [Ideas]
I was wondering if it was somehow possible to also customize the icon for the launcher?.

Currently my launcher widget just picks random letters for the icon, would be nice if I could have an image there instead.

![image](https://github.com/trungleduc/jupyter_app_launcher/assets/24872459/4a78b45f-a8f6-49da-9af8-eb95e4fab08f)

Amazing work.

**Comment by trungleduc** [comment]
Hi, you have the `icon` field in the config to set the icon of your app 
https://jupyter-app-launcher.readthedocs.io/en/latest/usage.html#configuration-file-content

**Reply by bijucyborg** [reply]
Thanks a ton for highlighting that.
